app/utility/payload_encoder.py

```diff
@@ -0,0 +1,65 @@
+"""
+This module contains helper functions for encoding and decoding payload files.
+
+If AV is running on the CALDERA server host, then it may sometimes flag, quarantine, or delete
+CALDERA payloads. To help prevent this, encoded payloads can be used to prevent AV
+from breaking CALDERA server-side. The convention expected by the CALDERA server is that
+encoded payloads will be XOR'ed with the DEFAULT_KEY contained in the payload_encoder.py
+module.
+
+Additionally, payload_encoder.py can be used from the command-line to add a new encoded payload.
+
+```
+python /path/to/payload_encoder.py /path/to/plain-text-file /path/to/encoded-file.xored
```

Comment:
tighten this up. the /path/to makes this look busier than it should. make this:
```
python payload_encoder.py input_file output_file 
```
Also remove the word 'CALDERA' from these comments

Reply:
f2be5fc1da4e4a40d398b7dc442bc448a7a0e524